io_scene_dae/export_dae.py

```diff
@@ -61,27 +61,26 @@ def snap_tup(tup):
 
 
 def strmtx(mtx):
-    s = " "
+    s = ""
     for x in range(4):
         for y in range(4):
-            s += str(mtx[x][y])
-            s += " "
-    s += " "
+            s += " {}".format(mtx[x][y])
+    s = " {} ".format(s)
     return s
```

Comment:
I'm not sure the above is strictly equivalent to the previous behaviour, though I don't know if it's of any relevance to the exporter. The previous string used to start with one whitespace, and end with two (" a b c d  "), while the new format will start with two whitespace and end with one ("  a b c d ").